sentry_streams/src/filter_step.rs

```diff
@@ -0,0 +1,66 @@
+use crate::routes::{Route, RoutedValue};
+use pyo3::{Py, PyAny, Python};
+use sentry_arroyo::processing::strategies::{
+    CommitRequest, ProcessingStrategy, StrategyError, SubmitError,
+};
+use sentry_arroyo::types::Message;
+use std::time::Duration;
+
+pub struct Filter<N> {
+    pub callable: Py<PyAny>,
+    pub next_step: N,
```

Comment:
I think this can be more specific: `Box<dyn ProcessingStrategy<RoutedValue>>` 

The next step has to be a ProcessingStrategy and we know we only pass around RoutedValue

Reply:
What's the reasoning for the `dyn` here?

Reply:
ProcessingStrategy is a trait, I think dyn is required.